packages/core/types/@chainlink/index.d.ts

```diff
@@ -107,6 +108,20 @@ declare module '@chainlink/types' {
     error: ErrorBasic | ErrorFull
   }
 
+  export type Telemetry = {
+    rateLimitEnabled: boolean
+    wsEnabled: boolean
+    cacheEnabled: boolean
+    cacheType?: string
+    cacheWarmingEnabled: boolean
+    cacheMaxAge?: string
+    metricEnabled: boolean
+    rateLimitApiTier?: string
+    requestCoalescingEnabled: boolean
+    protocol?: 'ws' | 'http'
```

Comment:
It seems like this metric might be too difficult to keep accurate.

Let's please remove this from the spec for now. I think if we were to add it now, it wouldn't serve its original purpose. This would be easier to implement in EAv3.

Reply:
oh sorry, didn't notice that